app/src/main/java/ch/sdp/vibester/activity/ProfileActivity.kt

```diff
@@ -172,21 +215,43 @@ class ProfileActivity : AppCompatActivity() {
                 dataGetter.setFieldValue(FireBaseAuthenticator().getCurrUID(), "username",  input.text.toString())
             }
         }
-
         builder.setNegativeButton("Cancel") { dialog, _ -> dialog.cancel() }
         builder.show()
     }
 
     /**
-     * A function that displays the dialog.
-     * @param id: The id of the user to be shown.
-     * @param name: The name of the user to be shown.
+     * A function shows an image change dialog.
+     * @param title title of the dialog
      */
-    private fun showGeneralDialog(id: Int, name: String) {
-        val title = "Create $name"
-        val hint = "Enter new $name"
 
-        showDialog(title, hint, 0, id, name)
+    private fun showImageChangeDialog(title: String) {
+        val builder: AlertDialog.Builder = AlertDialog.Builder(this)
+        builder.setTitle(title)
+
+        builder.setPositiveButton("Yes") { _, _ ->
+            dataGetter.getCurrentUser()?.let { updateImage(it.uid) }
+        }
+
+        builder.setNegativeButton("No") { dialog, _ -> dialog.cancel() }
+        builder.show()
+    }
+
+    /**
+     * A function shows a dialog.
+     * @param name name of the dialog
+     * @param textDialog boolean to check the type of dialog
+     */
+
+    private fun showGeneralDialog(name: String, textDialog: Boolean) {
+        if(textDialog) {

```

Comment:
Space after if

Reply:
what do you mean?

Reply:
a space between the if and the parenthesis